archivy/routes.py

```diff
@@ -121,15 +121,18 @@ def show_dataobj(dataobj_id):
                 if hit["id"] != dataobj_id:
                     backlinks.append({"title": hit["title"], "id": hit["id"]})
 
+    post_title_form = forms.TitleForm()
+    post_title_form.title.data = dataobj["title"]
+
     return render_template(
         "dataobjs/show.html",
-        title=dataobj["title"],
```

Comment:
The way the template works is the title attribute fills the `<title>` tag of the returned html, and this is true for all routes, so we need it. Don't worry, I'll add it back :)

Reply:
sure